### PR TITLE
feat(association-select): add selectable and placeholder value option

### PR DESCRIPTION
**Pull request by VMBindraban**

### CODE REVIEW

File: src/component/framework/association.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ export class Association {
 
   association;
   manyAssociation;
+  selectablePlaceholder = 'false';
```
Comment (bas080) [inline src/component/framework/association.js]: Would it possible to use a Boolean type `false`

File: src/component/framework/association.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ export class Association {
...
+  placeholderValue      = '0';
```
Comment (bas080) [inline src/component/framework/association.js]: Would it be possible to use a number.

**Comment by VMBindraban** [comment]
@bas080 Re-review plz.

File: src/component/framework/association.js [hunk 3]
```diff
@@ -5,6 +5,8 @@ export class Association {
 
   association;
   manyAssociation;
+  selectablePlaceholder = false;
```
Comment (bas080) [inline src/component/framework/association.js]: These should be defined on the `element` object.

File: src/component/framework/association.js [hunk 4]
```diff
@@ -23,6 +25,14 @@ export class Association {
     if (this.manyAssociation) {
       this.manyAssociation = this.formField.model[this.formField.element.manyAssociation];
     }
+
+    if (typeof element.selectablePlaceholder !== 'undefined') {
```
Comment (bas080) [inline src/component/framework/association.js]: Instead of these if statements you can bind them directly to the association-select.

**Comment by VMBindraban** [comment]
@bas080 Re-review plz.

**Review by bas080 — APPROVED** [review]
Looks good, @VMBindraban, have you tested it also?

**Comment by RWOverdijk** [comment]
@bas080 @VMBindraban is not in today. Could you test it?

**Comment by bas080** [comment]
@RWOverdijk, :ok_hand: testing now

**Comment by RWOverdijk** [comment]
@bas080 Awesome.

**Comment by bas080** [comment]
It is setting the value. However something unexpected is happening.

test

It's parsing the value to the string `"null"`

**Comment by RWOverdijk** [comment]
Not good.

**Comment by VMBindraban** [comment]
This behavior comes from aurelia-orm. 
Made a PR for it: SpoonX/aurelia-orm#257

File: src/component/framework/association.js [hunk 5]
```diff
@@ -5,6 +5,8 @@ export class Association {
 
   association;
   manyAssociation;
+  selectablePlaceholder = false;
+  placeholderValue      = 0;
```
Comment (ppvg) [inline src/component/framework/association.js]: These two values are never used.

Perhaps you meant to use them as default values in `activate()` ([line 30](https://github.com/SpoonX/aurelia-form/pull/104/files#diff-ad697cea9b9f791e98207d3b65d56cc0R30))?

```js
element.placeholderValue = this.placeholderValue || 0
```

Comment (RWOverdijk) [reply src/component/framework/association.js]: 👍

**Comment by VMBindraban** [comment]
@bas080 Re-review plz.

**Comment by VMBindraban** [comment]
@bas080 bump

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**